app/views/shared/vacancy/_jobseeker_view.html.haml

```diff
@@ -31,7 +31,7 @@
   .govuk-grid-column-two-thirds
 
     - if @vacancy.organisations.many?
-      = render(NotificationComponent.new(content: t('jobs.notifications.content.multi_school_job_notification_html', organisation_type: organisation_type_basic(@vacancy.parent_organisation)), style: 'notice', background: true, dismiss: false, alert: true))
+      = render(Shared::NotificationComponent.new(content: t('jobs.notifications.content.multi_school_job_notification_html'), style: 'notice', background: true, dismiss: false, alert: true))
```

Comment:
Why has the content here been changed?

Reply:
good spot, im not sure am reverting this